lisa/tools/lisdriver.py

```diff
@@ -0,0 +1,85 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from pathlib import PurePath
+from typing import List, Type
+
+from lisa.base_tools import Wget
+from lisa.executable import Tool
+from lisa.operating_system import Redhat
+from lisa.tools.tar import Tar
+from lisa.util import UnsupportedDistroException, UnsupportedKernelException
+from lisa.util.process import ExecutableResult
+
+from .modinfo import Modinfo
+
+
+class LisDriver(Tool):
+    """
+    This is a virtual tool to detect/install LIS (Linux Integration Services) drivers.
+    More info  - https://www.microsoft.com/en-us/download/details.aspx?id=55106
+    """
+
+    @property
+    def dependencies(self) -> List[Type[Tool]]:
+        return [Wget, Modinfo]
+
+    @property
+    def command(self) -> str:
+        return "modinfo hv_vmbus"
+
+    @property
+    def can_install(self) -> bool:
+        if (
+            isinstance(self.node.os, Redhat)
+            and self.node.os.information.version < "7.8.0"
+        ):
+            return True
+
+        raise UnsupportedDistroException(
```

Comment:
Based on the supports up to Redhat 7.8. Is it end of life? Can we remove the lisdriver tool?

Reply:
it is EOL, `7.8 (retired  September 30, 2020)`, but in our testing, sometimes we pick up the old distro for testing